lte/cloud/go/services/lte/servicers/builder_servicer.go

```diff
@@ -528,18 +528,21 @@ func getServiceAreaMaps(serviceAreaMaps map[string]lte_models.TacList) map[strin
 
 // getFederatedNetworkConfigs in case this is a federated LTE networkm this function will try to parse out
 // feg_models.FederatedNetworkConfigs out of it
-func getFederatedNetworkConfigs(fegId lte_models.FegNetworkID, request *builder_protos.BuildRequest) (*feg_models.FederatedNetworkConfigs, error) {
-	if fegId == "" {
+func getFederatedNetworkConfigs(networkType string, fegId lte_models.FegNetworkID, request *builder_protos.BuildRequest)(*feg_models.FederatedNetworkConfigs, error) {
+	if networkType != feg.FederatedLteNetworkType {
 		// this is a non federated network, return nothing
 		return nil, nil
 	}
+	if fegId == "" {
+		glog.Warning(fmt.Errorf("Cant get LTE Federated Configs since network is of type %s, " +
+			"but feg ID is not defined: %s. If this is non federated network, remove the `fed_netowrk id` field. " +
```

Comment:
I think the section `If this is non federated network, remove the `fed_netowrk id` field. ` is unnecessary as we're already checking network type.

Reply:
You are right. That doesn't make sense anymore.
Let me simplify the message